cephfs/cephfs.go

```diff
@@ -150,6 +235,16 @@ func (mount *MountInfo) Chmod(path string, mode uint32) error {
 	return nil
 }
 
+// Fchmod changes the mode bits (permissions) of an open file
+func (mount *MountInfo) Fchmod(fd int, mode uint32) error {
```

Comment:
Please add tests for all the new methods. I don't see a test for this, did I miss it somewhere?